src/xpcc/driver/storage/block_device_file.hpp

```diff
@@ -0,0 +1,97 @@
+// coding: utf-8
+/* Copyright (c) 2018, Raphael Lehmann
+ * All Rights Reserved.
+ *
+ * The file is part of the xpcc library and is released under the 3-clause BSD
+ * license. See the file `LICENSE` for the full license governing this code.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef XPCC_BLOCK_DEVICE_FILE_HPP
+#define XPCC_BLOCK_DEVICE_FILE_HPP
+
+#include <xpcc/processing/resumable.hpp>
```

Comment:
This and all other drivers should include `<xpcc/architecture/interface/block_device.hpp>` and inherit from it.